ee/api/test/test_integration.py

```diff
@@ -0,0 +1,48 @@
+import hashlib
+import hmac
+import json
+import time
+from typing import Any
+
+from ee.api.test.base import APILicensedTest
+from posthog.models.instance_setting import set_instance_setting
+
+
+class TestIntegration(APILicensedTest):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        set_instance_setting("SLACK_APP_SIGNING_SECRET", "not-so-secret")
+
+    def _headers_for_paylod(self, payload: Any):
```

Comment:
typo :)

Reply:
No it isn't. Paylod is the... lighter version of... payload... or something... 🤦‍♂️

Reply:
Ah lol